.all-contributorsrc

```diff
@@ -328,6 +328,15 @@
         "doc",
         "code"
       ]
+    },
+    {
+      "login": "mihirg008",
+      "name": "Mihir Gupta",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/32246337?s=460&u=26abf142607d487660a12fcc8b508407b0896c8c&v=4",
+      "profile": "https://github.com/mihirg008",
+      "contributions": [
+        "doc"
+      ]
```

Comment:
You can run the `npm run contributors:generate` command mentioned in the [Contributing document](https://github.com/CodeToExpress/dailycodebase/blob/master/CONTRIBUTING.md) to add your name to the [Contributors document](https://github.com/CodeToExpress/dailycodebase/blob/master/CONTRIBUTORS.md)